src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1269,6 +1270,24 @@ public function deleteProductAction(int $orderId, int $orderDetailId): JsonRespo
         }
     }
 
+    /**
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))", redirectRoute="admin_orders_index")
+     *
+     * @param int $orderId
+     *
+     * @return JsonResponse
+     */
+    public function getDiscountsAction(int $orderId): Response
+    {
+        /** @var OrderForViewing $orderForViewing */
+        $orderForViewing = $this->getQueryBus()->handle(new GetOrderForViewing($orderId));
+
+        return $this->render('@PrestaShop/Admin/Sell/Order/Order/Blocks/View/discount_list.html.twig', [
+            'discounts' => $orderForViewing->getDiscounts()->getDiscounts(),
+            'orderForViewingId' => $orderForViewing->getId(),
```

Comment:
`orderId` is enough here I think
Since it's only the ID it no longer carries any viewing info

Reply:
Fixed